src/pupil_labs/invisible_lsl_relay/xdf_cloud_time_sync.py

```diff
@@ -0,0 +1,70 @@
+import json
+import os.path
+import pathlib
+
+import click
+import pyxdf
+
+
+@click.command()
+@click.argument('path_to_xdf', nargs=1)
+@click.argument('paths_to_exports', nargs=-1)
+def main(path_to_xdf, paths_to_exports):
+    check_input(path_to_xdf, paths_to_exports)
+    load_files(path_to_xdf, paths_to_exports)
+
+
+def check_input(path_to_xdf, paths_to_export):
+    if not pathlib.Path(path_to_xdf).is_file():
+        # path_to_xdf should be a file with the ending .xdf
+        raise FileNotFoundError(f'The path {path_to_xdf} does not point to a file.')
```

Comment:
This kind of sanity check can be delegated to click https://click.palletsprojects.com/en/8.1.x/arguments/#file-path-arguments